Makefile

```diff
@@ -37,8 +37,9 @@ TEST_PACKAGES = $(filter-out $(IGNORED_PACKAGES),$(PACKAGES))
 TEST_IGNORED_PACKAGES = $(filter $(IGNORED_PACKAGES),$(PACKAGES))
 
 # Configure bine.
-include hack/make/tools.mk
 export PATH := $(shell go tool bine path):$(PATH)
+tool-%:
+	@go tool bine get $* 1> /dev/null
```

Comment:
I think the targets are sorted alphabetically in this Makefile, it would be nice if we could move this just above `tools`.

Reply:
I can do that! Thanks.